src/Turing.jl

```diff
@@ -138,5 +138,10 @@ export  @model,                 # modelling
         elementwise_loglikelihoods,
         generated_quantities,
         logprior,
-        logjoint
+        logjoint,
+
+        get_densities_and_θs,     # Tempering 
+        make_tempered_model,
+        make_tempered_logπ,
+        get_θ
```

Comment:
Is it necessary to export these functions?

Reply:
In particular, I am not happy about `get_θ` and `get_densities_and_θs`: the API should not rely on unicode characters (so there should be at least an ASCII alternative) and also the names seem quite general and not limited to tempering.

Reply:
Fair enough on unicode, wasn't aware of that design parameter, I will just rename to `get_params` and `get_tempered_densities_and_params`? As for the functionality, I agree its very general, and the former `get_params` function is certainly not limited to usage for tempering but is at least first implemented for this, is there somewhere else we could put it? Perhaps in the inference sub-module given it should act on any individual sampler offered by Turing?

Reply:
Then there is also a question of where to put `get_vi` which is similarly general but first required here it seems

Reply:
My main question is: where are these functions needed? Are users expected to call them?

Reply:
Additionally, we already have functions such as `params`, `metadata`, and `getlogevidence` for querying transitions.